### PR TITLE
Add logging charts to sync-charts

### PR DESCRIPTION
**Pull request by xrstf**
**What this PR does / why we need it**:
This PR adds the logging charts to the sync-charts script.

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Comment by mrIncompetent** [comment]
Looks good, but @alvaroaleman needs to review the bash parts here.

@xrstf please verify if https://github.com/kubermatic/docs/blob/master/content/installation/add_seed_cluser/_index.en.md needs an update as well

**Comment by xrstf** [comment]
> please verify if https://github.com/kubermatic/docs/blob/master/content/installation/add_seed_cluser/_index.en.md needs an update as well

It sure does, but I wanted to do that after ensuring this PR went in the right direction. :)

File: api/hack/sync-charts.sh [hunk 1]
```diff
@@ -14,69 +15,114 @@ EOF
...
+  commit="local folder"
 fi
 
-rm -rf ${TARGET_DIR}
-mkdir ${TARGET_DIR}
-git clone https://github.com/kubermatic/kubermatic-installer.git ${TARGET_DIR}
-cd ${TARGET_DIR}
-git checkout ${INSTALLER_BRANCH}
+# create fresh clone from the installer repo
+rm -rf "${target_dir}"
```
Comment (alvaroaleman) [inline api/hack/sync-charts.sh]: Please use upper case for variables in bash (everywhere, not just here)

**Review by alvaroaleman — COMMENTED** [review]
Well I must say I found it way easier to read before but if you are going to maintain this I will not block on that

File: api/hack/sync-charts.sh (hunk 1 above)
Comment (xrstf) [reply api/hack/sync-charts.sh]: Why, except for personal preference? They are not exported and/or environment variables, so it made more sense to me to keep them lowercase. I have no really strong opinions either way, but is there a good reason to uppercase every variable regardless of scope? :-)

**Comment by xrstf** [comment]
> found it way easier to read before

You found it easier to duplicate chunks of code that are 100% identical to each other? For adding the logging stuff, I would have to add a third block of

```
for CHART in ${LOGGING_CHARTS}; do
  echo "syncing ${CHART}..."
  # doing clean copy
  rm -rf ${TARGET_DIR}/charts/logging/${CHART}
  cp -r ${CHARTS_DIR}/logging/${CHART} ${TARGET_DIR}/charts/logging/${CHART}
  echo "# ====== ${CHART} ======" >> ${TARGET_VALUES_FILE}
  cat "${CHARTS_DIR}/logging/${CHART}/values.yaml" >> ${TARGET_VALUES_FILE}
  echo "" >> ${TARGET_VALUES_FILE}
done
```

And when you then want to finally start differentiating between masters and seed, you would have to duplicate everything once again, so we have the block above six times.

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**